packages/generate-newsletter/index.js

```diff
@@ -43,12 +43,13 @@ const askQuestions = async () => {
 askQuestions().then(userResponse => {
   console.log('\n Generating pls wait... \n ')
 
-  extractSheets({ spreadsheetKey, credentials }, (err, data) => {
+  extractSheets({ spreadsheetKey, credentials, sheetsToExtract: ['latest'] }, (err, data) => {
     if (err) {
       console.log('ERROR: ', err)
       return
     }
 
+    console.log('\n Received content :  \n ', JSON.stringify(data));
```

Comment:
Do we need this ??

Reply:
yeah.. just to make sure that we get the right content always when we generate it.